Pipelines/ci.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# CI build producing developer packages.
+
+variables:
+- template: Config/settings.yaml
+- name: Version.Revision
+  value: $[counter(variables['MRTKGraphicsToolsVersion'], 1)]
+- name: Version.PreReleaseRevision
+  value: $[counter(variables['MRTKGraphicsToolsReleaseTag'], 1)]
+
+resources:
+  repositories:
+  - repository: PipelineTools
+    type: git
+    endpoint: ToolsAccess
+    name: tools.internal
+    ref: 78ecf591964ecf511b4c7a0cd84d98ce11e8be16
+
+pr: none # disable PR validation
+trigger:
+  tags:
+    include:
+    - '*'
+  branches:
+    exclude:
+    - main
+    - user-tayomadein-ci-cd
+
+jobs:
+- job: UnityValidation
```

Comment:
Sorry I'm not familiar with this syntax. What does this validation step do?

Reply:
The "UnityValidation" is the name of the job, it isn't the best name based on the steps, however I was torn between renaming it and staying consistent. An ideal name will be "PackageAndPublish"